src/main/java/com/example/java/test/junior/developer/configuration/SecurityConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.example.java.test.junior.developer.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
`'org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter' is deprecated ` since Spring Boot 2.7.0
Please, read documentation, search the Internet on this topic. Refactor this class according to the latest Spring's recommendations.